src/javascript/app/pages/trade/markets.jsx

```diff
@@ -0,0 +1,441 @@
+            },
+        });
+
+        // Trigger change event.
+        // TODO: move this block to componentDidUpdate
+        this.$underlying.value = underlying_symbol;
+        this.$underlying.setAttribute('data-text',this.underlyings[underlying_symbol]);
+        const event = new Event('change');
+        this.$underlying.dispatchEvent(event);
+
+        setTimeout(this.closeDropdown, 500);
+        /* Todo add notification for closed markets */
+        // Notifications.show({ text: localize('All markets are closed now. Please try again later.'), uid: 'MARKETS_CLOSED' });
+
+    }
+
+    onTabChange = (e) => {
+        const market = e.target.dataset.market;
+        this.scrollToElement(`${market}_market`, 120, 0);
+    }
+
+    saveRef = (node_name, node) => this.references[node_name] = node;
+
+    scrollToElement = (id, duration = 120, offset = 0) => {
+        // handleScroll is triggered automatically which sets the active market.
+        const {list} = this.references;
+        const toOffset = getElementById(id).offsetTop - list.offsetTop - offset;
+        scrollToPosition(list, toOffset, duration);
+    }
+
+    stickyHeader = (position) => {
+        let curr, prev, next;
+        const {market_nodes} = this.references;
+        const market_keys = Object.keys(market_nodes);
+        const TITLE_HEIGHT = 40;
+        Object.values(market_nodes).forEach((node, idx) => {
+            if (node.dataset.offsetTop <= position
+                && +node.dataset.offsetHeight + +node.dataset.offsetTop > position) {
+                curr = node;
+                prev = idx > 0 ? market_nodes[market_keys[idx-1]] : null;
+                next = idx < market_keys.length ? market_nodes[market_keys[idx+1]] : null;
+            }
+        });
+
+        const class_sticky = 'sticky';
+        const class_under = 'put_under';
+        const DEFAULT_TOP = this.references.list.offsetTop;
+
+        if (curr) {
+            curr.children[0].removeAttribute('style');
+            curr.removeAttribute('style');
+            curr.children[0].classList.remove(class_under);
+            const diff = (+curr.dataset.offsetHeight + +curr.dataset.offsetTop) - position ;
+            if (diff > 0 && diff < TITLE_HEIGHT) {
+                curr.children[0].style.top = `${DEFAULT_TOP - (TITLE_HEIGHT - diff)}px`;
+                curr.children[0].classList.add(class_under);
+            }
+            curr.children[0].classList.add(class_sticky);
+            curr.style.paddingTop = `${TITLE_HEIGHT}px`;
+        }
+        if (prev) {
+            prev.removeAttribute('style');
+            prev.children[0].removeAttribute('style');
+            prev.children[0].classList.remove(class_under);
+            prev.children[0].classList.remove(class_sticky);
+        }
+        if (next) {
+            next.children[0].classList.remove(class_sticky);
+            next.children[0].classList.remove(class_under);
```

Comment:
you can remove multiple classes in the same line, e.g. `next.children[0].classList.remove(class_sticky, class_under)`. same for above